pandas/tests/resample/test_period_index.py

```diff
@@ -548,6 +548,19 @@ def test_quarterly_resampling(self):
         exp = ts.to_timestamp().resample("A").mean().to_period()
         tm.assert_series_equal(result, exp)
 
+    def test_period_resample_sum_min_count(self):
```

Comment:
can this test be done more generally in test_base?

Reply:
thanks for the feedback @jreback - moved the test and generalised to include tests for DTI, PI and TDI.